lib/phlex/rails/layout.rb

```diff
@@ -12,5 +12,37 @@ module Layout
 		include Helpers::JavaScriptIncludeTag
 		include Helpers::JavaScriptImportMapTags
 		include Helpers::JavaScriptImportModuleTag
+
+		module ClassMethods
+			def render(view, _locals, &block)
+				component = allocate
+
+				component.call(view_context: view) do |yielded|
+					case yielded
+					when Symbol
+						output = view.view_flow.get(yielded)
+					else
+						output = yield
+					end
+
+					case output
+					when ActiveSupport::SafeBuffer
+						component.unsafe_raw output
+					end
+				end
+			end
+
+			def identifier
+				name
+			end
+
+			def virtual_path
+				""
+			end
```

Comment:
I don’t know what a `virtual_path` is. It’s not really documented anywhere. All I know is the layout needs to respond to `virtual_path` and it must return a String. I *think* this has something to do with Rails keeping track of changes for reloading, which isn't necessary for Ruby files anyway.